nokia/sonic-platform
# [H4-64D]Add support for H4-64D platform.

### Why I did it
Add support for NOKIA 7220-IXR-H4-64D platform:

> Platform: x86_64-nokia_ixr7220_h4-r0
> HwSKU: Nokia-IXR7220-H4-64D
> ASIC: Broadcom
> Port Config: 64x400G + 2x10G

### How I did it
1.	Add platform related directory ixr7220h4-64d in platform/Broadcom/sonic-platform-modules-nokia
2.	Modified related build scripts in platform/Broadcom/sonic-platform-modules-nokia/debian
### How to verify it
1.	Make sure the sonic-buildimage is successful
2.	Run this image on x86_64-nokia_ixr7220_h4-r0 and verify all dockers are up and test basic commands like:
o show version
o show platform summary
o show platform syseeprom
o show platform fan
o show platform psustatus
o show platform firmware status
o show platform temperature
o sudo show system-health detail
o show interface status
3.	Run OC test with T1-64-lag topology and have 100% pass on platform.
